### PR TITLE
Fix falling and fallen

### PR DESCRIPTION
**Pull request by Flova**
## Proposed changes
- Dont interpret quaternion as euler angles in the `falling`
- Use fused angles instead of euler
- Use orientation instead of gyro to determine the `falling` direction if a fall is detected (less issues with bouncing robots)
- Use fused angles to determine the `fallen` detection
- Remove `hacky_sequence_dynup_running` which blocks falling very oftern in sim (maybe also on robot)

## Necessary checks
- [x] Update package version
- [x] Run `catkin build`
- [x] Test on your machine
- [x] Test on the robot
- [x] Put the PR on our Project board

### CODE REVIEW

**Review by SammyRamone — CHANGES_REQUESTED** [review]
please check if the "todo hip pitch offset" is resolved. I think that it is implicitly resolved by having different config parameters for front and back.

**Review by SammyRamone — APPROVED** [review]

**Comment by Flova** [comment]
> please check if the "todo hip pitch offset" is resolved. I think that it is implicitly resolved by having different config parameters for front and back.

This TODO is older than the current implementation and not needed anymore.

File: bitbots_hcm/src/bitbots_hcm/hcm_dsd/decisions/decisions.py [hunk 1]
```diff
@@ -365,7 +364,7 @@ def perform(self, reevaluate=False):
             return "NOT_FALLEN"
 
     def get_reevaluate(self):
-        return not self.blackboard.hacky_sequence_dynup_running
+        return True
```
Comment (timonegk) [inline bitbots_hcm/src/bitbots_hcm/hcm_dsd/decisions/decisions.py:367]:
```suggestion
        return True
```

**Review by timonegk — CHANGES_REQUESTED** [review]
Are you sure that `hacky_sequence_dynup_running` can be removed? It was once necessary to avoid that the robot detects fallen while executing dynup.

**Comment by SammyRamone** [comment]
tested it on the robot, works

**Comment by Flova** [comment]
> Are you sure that `hacky_sequence_dynup_running` can be removed? It was once necessary to avoid that the robot detects fallen while executing dynup.

It works. If the dynup needs adaptations it should be done in the dynup branch. IMO.

**Review by timonegk — APPROVED** [review]

**Pull request merged**